CIScripts/Deploy/Deployment.ps1

```diff
@@ -63,6 +63,9 @@ function Install-Artifacts {
         }
     }
 
+    Write-Host "Copying dlls"
+    Copy-Item -ToSession $Session -Path "dlls\*" -Destination "C:\Windows\System32\"
```

Comment:
I don't like that here we assume all the paths to artifacts.

In Build.ps1 we have all these
```

  $vRouterOutputDir = "output/vrouter"		  $vRouterOutputDir = "output/vrouter"
  $AgentOutputDir = "output/agent"		  $AgentOutputDir = "output/agent"
```
but now we don't use any variables like $vRouterOutputDir.

This is implicit global state.

IMHO we should move these paths to a config file in the future (not this PR) - will add this to backlog.

Reply:
Testing and deployment assumes location of all artifacts, so the best way would be for this to be configured so that Build, Deploy and Test uses the same locations. But this is not in scope for this PR.

Reply:
Agreed